sphinx_gallery/gen_gallery.py

```diff
@@ -317,6 +319,17 @@ def call_memory(func):
                               % (css, _KNOWN_CSS))
         if gallery_conf['app'] is not None:  # can be None in testing
             gallery_conf['app'].add_css_file(css + '.css')
+    if gallery_conf['thebelab'] is not None:
+        gallery_conf['thebelab']['selector'] = \
+            ".sphx-glr-code>:not(.sphx-glr-output)"
+        gallery_conf['thebelab']['outputSelector'] = ".sphx-glr-output"
+        gallery_conf['thebelab']['predefinedOutput'] = True
+        gallery_conf['app'].add_js_file(
+            None,
+            body=json.dumps(gallery_conf['thebelab']),
+            type="text/x-thebe-config")
```

Comment:
this is a reasonable way to do the configuration for now, it may get trickier if there is a future possibility people will want per-page configuration of Thebelab (e.g. if two different kernels are used in the same gallery). In that case we'd need to add the Thebelab configuration at page build time

Reply:
ah actually I just realized we might need to use custom thebelab config on each page, because we'll need to set the kernel path to point to whatever sub-directory the gallery page is in. Otherwise Thebelab will initialize the kernel in the root of the docs, which may break relative paths.